### DISCUSSION TITLE
Show user hovercard when hovering avatar

### DISCUSSION THREAD

**Discussion by kidonng** [forem/forem#18439] [Feature Requests]
Comparing to other websites like GitHub and Twitter, Forem has a peculiar hovercard trigger. You have to hover a user's name (clicking it just toggles the hovercard), and hovering their avatar does no effect.

I suggest showing user hovercard when hovering their avatar, as usually expected on other sites.

**Comment by pkfrank** [comment]
Thanks @kidonng this is a good request / note.  Let's definitely take this into consideration @amywtlin @anujbhavsar96
